pkg/opentelemetry-mapping-go/otlp/metrics/metrics_translator_test.go

```diff
@@ -114,17 +114,17 @@ func newTranslatorWithStatsChannel(t *testing.T, logger *zap.Logger, ch chan []b
 
 	attributesTranslator, err := attributes.NewTranslator(set)
 	require.NoError(t, err)
-	tr, err := NewTranslator(
+	mt, err := NewDefaultTranslator(
```

Comment:
nit: variable name doesn't need to be changed
```suggestion
	tr, err := NewDefaultTranslator(
```